### PR TITLE
Help/About: Initial pass at 6.3 About page

### PR DESCRIPTION
**Pull request by ryelle**
Here's the first pass of the About page, adding in the strings from [this google doc](https://docs.google.com/document/d/1vxA4NnFvtTkBUaTQNG0KGuspIJfCGJgQp4N9GhLXfaE/edit#heading=h.igkh6tjltu0v).

Screenshot in comment below.

Trac ticket: https://core.trac.wordpress.org/ticket/58067

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Comment by annezazu** [comment]
Thanks so much for kicking this off. I added in some alternatives to the first item since it's a bit lengthy to try to offer a condensed version. I think the current design works though for the most part but would like to see "direct actions" rephrased, along with `Sharpen your designs with updated tools` and `Create and sync Patterns`. We need to call out the reusable block change and that latter title feels awkward. Can you all handle the updates @DanSoschin @eidolonnight ?

**Comment by ryelle** [comment]
Latest commits update the page to [the content in this doc](https://docs.google.com/document/d/1vxA4NnFvtTkBUaTQNG0KGuspIJfCGJgQp4N9GhLXfaE/edit#heading=h.mgpohs61aahb), and update the page layout and some assets to match figma.

Screenshots

about-about

about-credits

about-freedoms

about-privacy

about-contributors

**Comment by peterwilsoncc** [comment]
@ryelle Asking the silly question just in case: are the blue placeholders on the new contribute page to be replaced with graphics? If not, I think they'll need to be darker as they look a little WP 6.2.

**Comment by ryelle** [comment]
>  are the blue placeholders on the new contribute page to be replaced with graphics

They are supposed to be replaced, yes — I [reopened that ticket](https://core.trac.wordpress.org/ticket/23348#comment:116) so we can get those new graphics made.

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -55,162 +55,183 @@
...
 			</div>
 		</div>
 
 		<div class="about__section has-2-columns">
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Manage your menu in more ways with the Navigation block' ); ?></h3>
-				<p><?php _e( 'A new sidebar experience makes it easier to edit your site&#8217;s navigation. Add, remove, and reorder menu items faster—no matter how complex your menus are.' ); ?></p>
+				<h3><?php _e( 'Create and sync patterns' ); ?></h3>
+				<p><?php _e( 'Arrange Blocks in unlimited ways and save them as Patterns for use throughout your site. You can even specify whether to sync your patterns (previously referred to as “reusable blocks”) so that one change applies to all parts of your site. Or, utilize patterns as a starting point with the ability to customize each instance.' ); ?></p>
```
Comment (richtabor) [inline src/wp-admin/about.php]: "Blocks" should not be capitalized.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -55,162 +55,183 @@
...
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23dcdcde' /%3E%3C/svg%3E" alt="" />
 				</div>
 			</div>
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Discover a smoother experience for the Block Inserter' ); ?></h3>
-				<p><?php _e( 'A refreshed design gives you more visibility and easier access to the content you need. Use the Media tab to drag and drop content from your existing Media Library quickly. Find patterns faster with a split view that lets you navigate categories and see previews all at once.' ); ?></p>
+				<h3><?php _e( 'Work faster with the Command Palette' ); ?></h3>
+				<p><?php _e( 'Switch to a specific template or open your editor preferences with a new tool that helps you quickly navigate expanded functionality. With simple keyboard shortcuts (⌘+k on Mac or ctrl+k on Windows), clicking the sidebar search icon in Site View, or clicking the Edit View Title Bar while in Edit View, get where you need to go and do what you need to do in seconds, either in the Site Editor or the Post Editor.' ); ?></p>
```
Comment (richtabor) [inline src/wp-admin/about.php]: Replace "Edit View Title Bar with in Edit View" with "Title Bar"

Comment (richtabor) [reply src/wp-admin/about.php]: And remove ", either in the Site Editor or the Post Editor".

**Review by richtabor — CHANGES_REQUESTED** [review]

File: src/wp-admin/about.php [hunk 3]
```diff
@@ -55,162 +55,183 @@
...
 				<div class="about__image">
-					<img src="https://s.w.org/images/core/6.2/about-site-editor.png" alt="" />
+					<img src="data:image/svg+xml,%3Csvg width='436' height='436' viewbox='0 0 100 100' xmlns='http://www.w3.org/2000/svg'%3E%3Crect width='100%25' height='100%25' fill='%23dcdcde' /%3E%3C/svg%3E" alt="" />
 				</div>
 			</div>
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Explore and edit your site from the Site Editor' ); ?></h3>
-				<p><?php _e( 'An updated interface gives you more control over your site editing experience. Browse through full previews of your templates and template parts, then jump into editing your site from wherever you choose.' ); ?></p>
+				<h3><?php _e( 'Do everything in the Site Editor' ); ?></h3>
+				<p><?php _e( 'WordPress 6.3 brings your content, templates, and patterns together in the Site Editor for the first time. Add pages, browse style variations, create synced patterns, and enjoy fine-tuned control over your navigation menus. No more time wasted switching across different site areas—now you can focus on what matters most. Creation to completion, all in one place.' ); ?></p>
```
Comment (dream-encode) [inline src/wp-admin/about.php:71]: "site areas" sounds... weird.  Maybe "screens" or "pages"?

File: src/wp-admin/about.php [hunk 4]
```diff
@@ -55,162 +55,183 @@
...
+						<rect width="48" height="48" rx="4" fill="#151515"/>
+						<path d="M25.7781 16.8569L25.8 22.8573L28.9984 22.8572C29.805 22.8572 30.2796 23.6339 29.8204 24.2024L23.8213 31.6292C23.2633 32.3201 22.2013 31.9819 22.2013 31.1416L22.2 25.1481H19.0016C18.1961 25.1481 17.7212 24.3733 18.1782 23.8047L24.1496 16.3722C24.7055 15.6804 25.7749 16.0169 25.7781 16.8569Z" fill="white"/>
+					</svg>
 				</div>
-				<h3><?php _e( 'Focus on writing with Distraction Free mode' ); ?></h3>
-				<p><?php _e( 'For those times you want to be alone with your ideas. You can now hide all your panels and controls, leaving you free to bring your content to life.' ); ?></p>
+				<h3 style="margin-top:calc(var(--gap) * 0.75);margin-bottom:calc(var(--gap) * 0.5)"><?php _e( 'Performance gets a boost' ); ?></h3>
+				<p><?php _e( 'WordPress 6.3 has 170+ performance updates, including defer and async support for the WP Scripts API and fetchpriority support for images. These improvements can improve your website&#8217;s load time as perceived by visitors, along with block template resolution, image lazy-loading, and the emoji loader.' ); ?></p>
```
Comment (dream-encode) [inline src/wp-admin/about.php]:
```suggestion
				<p><?php _e( 'WordPress 6.3 has 170+ performance updates, including defer and async support for the Scripts API and fetchpriority support for images. These improvements can improve your website&#8217;s load time as perceived by visitors, along with block template resolution, image lazy-loading, and the emoji loader.' ); ?></p>
```

File: src/wp-admin/about.php [hunk 5]
```diff
@@ -55,162 +55,183 @@
...
+						<rect width="48" height="48" rx="4" fill="#151515"/>
+						<path d="M15 17.5H26V19H15V17.5Z" fill="white"/>
+						<path d="M18.5 23H29.5V24.5H18.5V23Z" fill="white"/>
+						<path d="M33 28.5H22V30H33V28.5Z" fill="white"/>
 					</svg>
 				</div>
-				<h3 class="is-smaller-heading" style="margin-top:0;margin-bottom:calc(var(--gap) / 4);"><?php _e( 'Importing widgets' ); ?></h3>
-				<p><?php _e( 'Options to import your favorite widgets from Classic themes to Block themes.' ); ?></p>
+				<h3 class="is-smaller-heading" style="margin-top:calc(var(--gap) / 2);margin-bottom:calc(var(--gap) / 4);"><?php _e( 'List View improvements' ); ?></h3>
+				<p><?php _e( 'Drag and drop to every content layer and delete any block you would like in the updated List View.' ); ?></p>
```
Comment (dream-encode) [inline src/wp-admin/about.php:215]:
```suggestion
				<p><?php _e( 'Drag and drop to every content layer and delete any block in the updated List View.' ); ?></p>
```

File: src/wp-admin/about.php (hunk 1 above)
Comment (ryelle) [reply src/wp-admin/about.php]: Done:

Screenshot 2023-07-18 at 11 11 32 AM

**Review by dream-encode — CHANGES_REQUESTED** [review]

File: src/wp-admin/about.php (hunk 2 above)
Comment (ryelle) [reply src/wp-admin/about.php]: Done:

Screenshot 2023-07-18 at 11 11 47 AM

**Comment by DanSoschin** [comment]
Unless there's a material error (typographical or factual), the About Page is closed for feedback. We've had more than enough sets of eyes on it, so it's ready to ship. Acknowledging that how features are presented and communicated is often a personal preference of style, the doc contributors have made an effort to keep this document brief and consistent with the WordPress voice and styles.

File: src/wp-admin/about.php (hunk 2 above)
Comment (richtabor) [reply src/wp-admin/about.php]: Thanks!

**Comment by ryelle** [comment]
Committed in [r56263](https://core.trac.wordpress.org/changeset/56263)

**Pull request closed without merging**